src/app/contexts/Ledger/hooks/connection.ts

```diff
@@ -35,10 +35,11 @@ export const useLedgerConnection = (transport: typeof Transport) => {
 	const importLedgerWallets = useCallback(
 		async (wallets: LedgerData[], coin: Coins.Coin, profile: Contracts.IProfile) => {
 			for (const { address, path } of wallets) {
-				const wallet = await profile.walletFactory().fromAddress({
+				const wallet = await profile.walletFactory().fromAddressWithLedgerPath({
 					address,
 					coin: coin.network().coin(),
 					network: coin.network().id(),
+					path,
 				});
 
 				wallet.data().set(Contracts.WalletData.LedgerPath, path);
```

Comment:
This can be removed when using `fromAddressWithLedgerPath`.

Reply:
Seems that `path` is required prop in `IAddressWithLedgerPathOptions` :hmm

Reply:
@goga-m this line `wallet.data().set(Contracts.WalletData.LedgerPath, path);`, as it is redundant